app/plugins/auth.plugin.js

```diff
@@ -47,6 +47,7 @@ const AuthPlugin = {
       },
       redirectTo: '/signin',
       validate: async (_request, session) => {
+        console.log('🚀 ~ file: auth.plugin.js:50 ~ validate: ~ session:', session)
```

Comment:
Doh! Something I should have spotted the first time around.

```suggestion
```

Reply:
weird that it's showing up on my PR.. it's not something I added. but i've removed it now